src/components/Content.js

```diff
@@ -0,0 +1,168 @@
+    var params = { brand: brands, searchString:searchString};
+    var url = '?' + ( new URLSearchParams( params ) ).toString();
+    
+    fetch("http://3.86.87.11:8080/getProducts/" + page + url)
+      .then(response => {
+        return response.json()
+      })
+      .then(data => {
+        setProducts(data);
+      })
+      .catch(err => {
+        console.log(err.message);
+      })
+
+      return () => {
+        // cleanup here
+      };
+  }
+  
+  useEffect(() => {
+    fetchProductsData(1,{},"");
+    setPageNo(1);
+  },[]);
+
+  useEffect(() => {
+    fetchProductsData(1,filters,searchString);
+    setPageNo(1);
+    
+    if(searchString !== ''){  
+      setResultMessage("Showing results for '" + searchString + "' with below filters.");
+    } else {
+      setResultMessage("Showing producs with below filters.");
+    }
+    
+  },[searchString,filters]);
+
+  useEffect(() => {
+    fetchProductsData(1,filters,searchString);
+  },[filters,searchString]);
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [pageNo]);
+
+  
+  
+
+  return (
+    
+    <>
+      <div className="container-fluid mt-5 py-4 px-4">
+        <nav aria-label="breadcrumb" className="bg-custom-light">
+          <ol className="breadcrumb p-3 mb-0">
+            {resultMessage}
+          </ol>
+        </nav>
+        <div className="row mb-0 mt-lg-3">
+          <div className="col-lg-2">
+            <div className="border shadow-sm flex-column">
+                <FilterView updateFilters = {setFilters}/>
+            </div>
+          </div>
+          <div className="col-lg-10">
+            {products.length > 0 ?
+            <>
+            <div className="d-flex flex-column">
+              <ProductsView cart={cart} setCart={setCart} products={products}/>
+            </div>
+            <div className="d-flex align-items-center mt-auto">
+              <span className="text-muted small d-none d-md-inline">
+                Showing {products.length} of 1224
+              </span>
+              <nav aria-label="Page navigation example" className="ms-auto">
+                <ul className="custom-page-bar pagination my-0">
+                  <li className="page-item">
+                  <button className="page-link" onClick={()=> {setPageNo((prev)=>prev-1)}}>
+                      Previous
+                    </button>
+                  </li>
+                  <li className="page-item active">
+                  <button className="page-link" onClick={()=> {}}>
+                      {pageNo}
+                    </button>
+                  </li>
+                  <li className="page-item">
+                  <button className="page-link" onClick={()=> {setPageNo((prev)=>prev+1)}}>
+                    {pageNo+1}
+                    </button>
+                  </li>
+                  <li className="page-item">
+                  <button className="page-link" onClick={()=> {setPageNo((prev)=>prev+2)}}>
```

Comment:
@rahulmeghwal it seems like this might show regardless of which page the customer is on (obviously it doesn't make sense to have a next button on the last page). I might be misreading the logic though. What's the best way to workaround that (especially given having +1 and +2 buttons)?

Reply:
Yeah, the page selector was not implemented earlier fully. I have now implemented this feature fully and updated the code.